src/api/ParticipantManager.Shared/Client/FeatureFlagClient.cs

```diff
@@ -2,22 +2,15 @@
 
 namespace ParticipantManager.Shared.Client;
 
-public class FeatureFlagClient : IFeatureFlagClient
+public class FeatureFlagClient(IFlagsmithClient flagsmithClient) : IFeatureFlagClient
 {
-    private readonly FlagsmithClient _flagsmithClient;
-
-    public FeatureFlagClient()
-    {
-        _flagsmithClient = new(Environment.GetEnvironmentVariable("FLAGSMITH_SERVER_SIDE_ENVIRONMENT_KEY"));
```

Comment:
this needs replacing with some code in the Program.cs files to register an IFlagSmithClient instance with the DI container